pytorch/training/docker/2.0/py3/Dockerfile.cpu

```diff
@@ -185,12 +188,17 @@ ARG PT_TORCHAUDIO_URL
 ARG PT_TORCHDATA_URL
 
 # Install ec2 AWS-PyTorch
-RUN /opt/conda/bin/mamba install -y pytorch=${PYTORCH_VERSION} \
+RUN /opt/conda/bin/mamba install -y pytorch=${PYTORCH_VERSION_EC2} \
   torchvision torchaudio torchtext cpuonly \
   --override-channels \
-  -c https://aws-ml-conda-pre-prod-ec2.s3.us-west-2.amazonaws.com \
+  -c https://aws-ml-conda-ec2.s3.us-west-2.amazonaws.com \
   -c conda-forge
 
+# Patches
+RUN pip install "pillow>=9.5"
+RUN /opt/conda/bin/mamba install -y -c conda-forge \
+  requests==2.31.0
```

Comment:
same goes for requests [here](https://github.com/aws/deep-learning-containers/blob/8cc613c2835def9335eb4df19d1489e16d6675f9/pytorch/training/docker/2.0/py3/Dockerfile.cpu#L117)